src/pki/est/est_tls.c

```diff
@@ -193,6 +193,19 @@ static bool parse_http_header(chunk_t *in,  u_int *http_code, u_int *content_len
 	return (*http_code < 300);
 }
 
+static bool is_whitespace(char input)
+{
+	switch (input)
+	{
+		case ' ':
+		case '\n':
+		case '\r':
+		case '\t':
+			return true;
+		default:
+			return false;
+	}
+}
```

Comment:
This is not necessary, there is a standard C function called `isspace()` for this purpose (you need to add `#include <ctype.h>` for the declaration).

Reply:
Sorry, I've missed that as I was thinking that the `isspace` from the first comment should show that it would be cleaner to read/understand when using a separate function with a clean name.

Updated.

Reply:
I should have pointed that out, sorry. Is it OK for you if I add myself as co-author to the commit?